langServer/tstValidation.ts

```diff
@@ -48,12 +50,17 @@ export function validateTextDocument(textDocument: TextDocument) {
   let withinValueUserCode: boolean = false;
   let withinExpectedUserCode: boolean = false;
   let withinImportFailures = false;
+  const codedTestsDriverInSubprogram = checkForKeywordInLine(
+    text,
+    "TEST.SUBPROGRAM",
+    "coded_tests_driver"
+  );
 
-  for (let lineIndex = 0; lineIndex < lineList.length; lineIndex++) {
+  for (lineIndex = 0; lineIndex < lineList.length; lineIndex++) {
     let thisLine: string = lineList[lineIndex];
```

Comment:
Not suggesting to do this rignt now, but line 61 - line 170 could be refactored into clearer statements with less nesting, such that it's clearer which particular case is being handled. Could be turned into an "internal issue" for later

Reply:
It would make sense to cover the currently uncovered lines in this range with unit tests

Reply:
Brought unit tests back to 100% in 265346f47f3d1a2bda59ea03cf5c5e455c889ea0